blueocean-core-js/src/js/fetch.js

```diff
@@ -290,7 +290,10 @@ function getPrefetchedDataFuture(url) {
                 // the prefetched data as the value.
                 if (trimRestUrl(preFetchEntry.restUrl) === trimmedUrl) {
                     try {
-                        return Promise.resolve(JSON.parse(preFetchEntry.data));
+                        const parse = JSON.parse(preFetchEntry.data);
+                        return Promise.resolve(parse);
+                    } catch (e) {
+                        console.error('preFetch Error', e);
```

Comment:
hmm interesting, I just reverted the core-js changes and only left the dashboard one and now I cannot see the problem anymore. It may be really a problem of faulty sse-event markup.